Controller/ResourceController.php

```diff
@@ -44,6 +44,9 @@ public function resourceAction($repositoryName, $path)
             $context
         );
 
-        return new Response($json);
+        $response = new Response($json);
+        $response->headers->set('Content-Type', 'application/json');
+        
+        return $response;
```

Comment:
No, as JsonResponse accepts an array to serialize, while wr serialize ourselves with the jms serializer.